src/components/Project/Releases/index.js

```diff
@@ -588,7 +597,7 @@ export default class Releases extends React.Component {
   }
 
   renderDrawer(){
-		if (this.state.drawerRelease === null){
+		if (this.state.drawerRelease == null){
```

Comment:
Why the change here?

Reply:
mistake, i was creating 2 state props so drawerRelease was being ignored. fixed now